### PR TITLE
Feat/contact edit

### PR DESCRIPTION
**Pull request by hcfw007**
#### Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added
- [ ] CI has been passed. (GitHub actions all turns green)
- [ ] CLA has been signed

### CODE REVIEW

File: proto/wechaty/puppet.proto [hunk 1]
```diff
@@ -241,6 +241,13 @@ service Puppet {
     };
   }
 
+  rpc ContactPayloadModify (puppet.ContactPayloadModifyRequest) returns (puppet.ContactPayloadModifyResponse) {
+    option (google.api.http) = {
+      put: "/contacts/{id}/payloadModify"
```
Comment (su-chang) [inline proto/wechaty/puppet.proto:246]: payloadModify 这个驼峰需要调整么？

File: proto/wechaty/puppet/contact.proto [hunk 2]
```diff
@@ -130,3 +130,27 @@ message ContactDeleteRequest {
 }
 
 message ContactDeleteResponse {}
+
+message ContactPayloadModifyRequest {
```
Comment (su-chang) [inline proto/wechaty/puppet/contact.proto:134]: code style

File: proto/wechaty/puppet.proto (hunk 1 above)
Comment (hcfw007) [reply proto/wechaty/puppet.proto:246]: 已修改

File: proto/wechaty/puppet/contact.proto (hunk 2 above)
Comment (hcfw007) [reply proto/wechaty/puppet/contact.proto:134]: 这个的问题没看懂

**Review by su-chang — APPROVED** [review]

**Pull request merged**